proto/sf/substreams/rpc/v2/service.proto

```diff
@@ -130,6 +130,37 @@ message OutputDebugInfo {
 
 message ModulesProgress {
   repeated ModuleProgress modules = 1;
```

Comment:
I think we shoul deprecate module progress right away

Reply:
ok, it makes sense. Not many clients will be parsing it anyway, except the GUI, and we'll change it at the same time. noted.